README.md

```diff
@@ -27,9 +27,9 @@ contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additio
 pass-thru:
   - model-deduplicator
   - subset-reducer
-version: ^3.1.0
+version: ~3.3.0
 use-extension:
-  "@autorest/modelerfour": ^4.15.456
+  "@autorest/modelerfour": ~4.19.1
```

Comment:
I would double check there is not issue with the security model added in 4.19.0 and the deprecated property on objects
https://github.com/Azure/autorest/blob/docs/feature-tracking/docs/developer/feature-status-tracking.md#feature-status-tracking

Also when this is merged I guess I can update the PR with at least partial support for `AnyObject` vs `Any`, Don't know if you have a plan to make a difference latter(make sure primitive types are not allowed whne yusing anyobject)

Reply:
hey @timotheeguerin, from what i can tell they won't break because those are additive changes

talked to @lmazuel offline, and it seems we want to treat "any" and "any-object" the same in python (reasoning being that everything's an object in python)